### DISCUSSION TITLE
Automate: Value of response from custom query

### DISCUSSION THREAD

**Discussion by Bernie1502** [Budibase/budibase#3799] [Help] [answered]
Hi,
i am testing the Automate-Feature.
When a row is created, i want to update the same row with some calculated data (from other tables) in a column.
So i do a custom query, with this repsonse:
![2021-12-15 21_38_39-Budibase und 1 weitere Seite - Geschäftlich – Microsoft​ Edge](https://user-images.githubusercontent.com/49621100/146261699-feda2f8f-7f3d-4ff1-b9ff-8f06c9cc213d.png)

Next Step is to update the row, also with a custom SQL-Query - the UpdateRow Feature didn't work for me...
I want to store the value of "Kurzbezeichnung" from the previous step in the column "BV_Nummer". How can i get only the value from the response? `{{steps.1.response.Kurzbezeichnung }}` didn't work.

![2021-12-15 21_42_29-Budibase und 1 weitere Seite - Geschäftlich – Microsoft​ Edge](https://user-images.githubusercontent.com/49621100/146262043-e6caa9a3-b8d3-4853-9ed9-51b2f1e515b8.png)

Thanks!

**Comment by mjashanks** [comment, marked as answer]
Hi @Bernie1502 

I think you need to use `{{ steps.1.response.0.Kurzbezeichnung }}`. `response` is an array, therefore you need to select the first value - i.e. `response.0`.

**Reply by Bernie1502** [reply]
Thx, that's it!
